WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -2171,6 +2175,36 @@ - (void)handleDataModelChange:(NSNotification *)note
     }
 }
 
+- (void)handleWillEnterForegroundNotification:(NSNotification *)note
+{
+    [self configureTableViewData];
+    [self reloadTableViewPreservingSelection];
+}
+
+- (void)observeManagedObjectContextObjectsDidChangeNotification
+{
+    NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(handleDataModelChange:)
+                                                 name:NSManagedObjectContextObjectsDidChangeNotification
+                                               object:context];
+}
```

Comment:
This change is not related to this PR but I moved this code into its own method to keep it consistent with `observeWillEnterForegroundNotification`.